From 99e2d400ad6ad7fbf4e2ad1a4277521eaad46d28 Mon Sep 17 00:00:00 2001
From: Craig Bradney <mrb@scribus.info>
Date: Sun, 4 Jun 2023 21:31:53 +0000
Subject: [PATCH 05/13] Rename m_doc in pdf_analyser to avoid confusion with
 m_doc in other code

git-svn-id: svn://scribus.net/trunk/Scribus@25495 11d20701-8431-0410-a711-e3c959e3b870
(cherry picked from commit b5aeed8093e06cf3df456f5ad26f76f6ca442bff)
---
 scribus/pdf_analyzer.cpp | 12 ++++++------
 scribus/pdf_analyzer.h   |  2 +-
 2 files changed, 7 insertions(+), 7 deletions(-)

--- a/scribus/pdf_analyzer.cpp
+++ b/scribus/pdf_analyzer.cpp
@@ -75,11 +75,11 @@ PDFAnalyzer::PDFAnalyzer(QString & filen
 		nameMapInited = true;
 	}
 
-	m_doc = nullptr;
+	m_pdfdoc = nullptr;
 
 	PdfError::EnableDebug( false );
 	try {
-		m_doc = new PdfMemDocument(filename.toLocal8Bit().data());
+		m_pdfdoc = new PdfMemDocument(filename.toLocal8Bit().data());
 	}
 	catch (PdfError & e)
 	{
@@ -91,14 +91,14 @@ PDFAnalyzer::PDFAnalyzer(QString & filen
 
 PDFAnalyzer::~PDFAnalyzer()
 {
-	delete m_doc;
+	delete m_pdfdoc;
 }
 
 bool PDFAnalyzer::inspectPDF(int pageNum, QList<PDFColorSpace> & usedColorSpaces, bool & hasTransparency, QList<PDFFont> & usedFonts, QList<PDFImage> & imgs)
 {
-	if (!m_doc)
+	if (!m_pdfdoc)
 		return false;
-	PdfPage* page = m_doc->GetPage(pageNum);
+	PdfPage* page = m_pdfdoc->GetPage(pageNum);
 	return page?inspectCanvas(page, usedColorSpaces, hasTransparency, usedFonts, imgs):false;
 }
 
@@ -725,7 +725,7 @@ void PDFAnalyzer::inspectExtGStateObj(Pd
 		if (arr[0].IsReference())
 		{
 			PdfReference ref = arr[0].GetReference();
-			PdfObject* fontObject = m_doc->GetObjects().GetObject(ref);
+		PdfObject* fontObject = m_pdfdoc->GetObjects().GetObject(ref);
 			if (fontObject)
 			{
 				PDFFont font = getFontInfo(fontObject);
--- a/scribus/pdf_analyzer.h
+++ b/scribus/pdf_analyzer.h
@@ -162,7 +162,7 @@ public:
 #ifdef HAVE_PODOFO
 private:
 	// pointer to the PoDoFo Pdf's object
-	PoDoFo::PdfMemDocument* m_doc;
+	PoDoFo::PdfMemDocument* m_pdfdoc;
 
 	// Path to the analyzed file
 	QString m_filename;
